docs/pages/eas-update/deployment.mdx

```diff
@@ -111,13 +111,17 @@
 
 ## Deploying to production
 
-When creating a new production build, increment your app version every time to ensure it has a unique runtime version.
+Run `eas update --channel production` to bundle and push a new update to production. This will make your hotfix immediately available to users of production builds with the same runtime version.
+
+**If you have already published the fix to staging and verified it there**, ensure that you are republishing from the same commit.
+
+We recommend using identical environment variables and code signing configuration on staging as on production, in order to ensure that updates verified in staging work exactly the same in production. If do this, then you can `eas update:republish --destination-channel production` to promote the update rather than generating a new one. This will ensure the exact same bundle that you tested in staging is used in production.
```

Comment:
I think it might be important to ensure people understand that once they vary their environment variables based on environment (which is common I think?) or change their signing in the same manner, they no longer can use this simplified release process.

```suggestion
For this release process we recommend using identical environment variables and code signing configuration on staging as on production, in order to ensure that updates verified in staging work exactly the same in production. If do this, then you can `eas update:republish --destination-channel production` to promote the update rather than generating a new one. This will ensure the exact same bundle that you tested in staging is used in production.
```